packages/reaction-email-templates/README.md

```diff
@@ -10,19 +10,28 @@ Provides functionality to load local default email templates, or optionally load
 Typical use of the exported `ReactionEmailTemplate`:
 
 ```js
-SSR.compileTemplate("<template>", ReactionEmailTemplate("<template>"));
+const html = ReactionEmailFromTemplate(
+  "accounts/inviteShopMember",
+  user.profile.language,
+  {
+    homepage: Meteor.absoluteUrl(),
+    shop: shop,
+    currentUserName: currentUserName,
+    invitedUserName: name,
+    url: Accounts.urls.enrollAccount(token)
+  }
+);
+
 try {
   return Email.send({
     to: email,
     from: `${shop.name} <${shop.emails[0].address}>`,
-    subject: `${shop.name} Update`,
-    html: SSR.render("<template>", {
-      homepage: Meteor.absoluteUrl(),
-      shop: shop,
-      order: order,
-      shipment: shipment
-    })
+    subject: `You have been invited to join ${shop.name}`,
+    html: html
   });
+} catch (_error) {
```

Comment:
this `_error` should be just `error`.  I think this was a remnant from when we converted from CoffeeScript.